### DISCUSSION TITLE
Descuento Bonus

### DISCUSSION THREAD

**Discussion by Cristian1754** [IIC2133-PUC/2024-1#100] [Tarea 1] [answered]
Una duda sobre el descuento de 5 decimas del bonus. Ese descuento se aplicara si el codigo se cae? O si no cumple con el peor caso log(n). Y lo otro ¿El criterio para ver si cumplio con el bonus sera el tiempo y una mirada rapida al codigo? Al menos eso habia entendido.

Saludos!

**Comment by sebacarrascop** [comment, marked as answer]
Claro, si el código se cae se aplica el descuento, y también si es que se demora más de los 10s máximos esperados de ejecución ya que para dar puntaje por bonus requerirá una revisión adicional. El criterio es que efectivamente se cumple la complejidad (en caso de no cumplirla, no se descontará si es que se cumple lo mencionado anteriormente), eso puede ser ya sea por tiempo o revisando manualmente el algoritmo.
